app/components/publications/publication/case/link-mandatee-modal.js

```diff
@@ -0,0 +1,72 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import { task } from 'ember-concurrency-decorators';
+
+/**
+ * @argument {PublicationFlow} publicationFlow (publication-flow,publication-flow.mandatees)
+ * @callback {() => Promise} onClose
+ * @callback {(mandatee: Mandatee) => Promise} onLink
+ * @dependsOn {Mandatee[]} mandatees ('mandatee,mandatee.person')
+ */
+export default class PublicationsPublicationCaseLinkMandateeModalComponent extends Component {
+  @service store;
+
+  @tracked options;
+  @tracked selection;
+
+  constructor() {
+    super(...arguments);
+
+    this.loadData();
+  }
+
+  get isLoading() {
+    return this.init.isRunning || this.onLink.isRunning;
+  }
+
+  @task
+  *init() {
+    yield this.loadData();
+    this.search('');
+  }
+
+  async loadData() {
+    const mandatees = this.store.peekAll('mandatee');
+    this.options = mandatees.filter((mandatee) => !this.isLinked(mandatee));
```

Comment:
Zie ook de bemerking in `publications/route.js`.  Met `peekAll` ga je niet alle mandatarissen opgehaald hebben in de frontend omdat dat veel kunnen zijn. Het filteren moet dan ook gebeuren met een filter in een backend request. Zie ook de `Utils::MandateeSelector` component.

Reply:
Het is niet gespecifieerd, maar wel logisch dat het op naam filtert: functietitel wordt niet (meer) getoond in het design, dus ik zou er ook niet aan denken erop te zoeken. Dan zou ik nog eerder enkel op achternaam filteren (voorlopig).

Reply:
Aha, die development-versie had ik nog niet opgemerkt :+1: 

De plaats waar de `Utils::MandateeSelector` component momenteel gebruikt wordt, is bij het toevoegen van een minister op een agendapunt. Hiervoor is [het prototype](http://development.kaleidos-prototype.mono.digital/shared/agendas/detail/dossier--minister-add-expanded.html) gelijkaardig aan de publicaties en dus niet in lijn met de huidige implementatie. Ik zou daarom de  `Utils::MandateeSelector` component  aanpassen zodanig dat het dan op beide plaatsen ineens correct is.